client/src/components/BenefitMatrix.ts

```diff
@@ -0,0 +1,90 @@
+export default class BenefitMatrix {
```

Comment:
I'm very unsure about this. Is there a nicer way to create these kind of data holding objects in JS? In Kotlin, this file could have been a three liner (one line per class).
Also not sure what the proper structuring for Vue projects is. Right now, this is in the components folder, which seems off.